### ISSUE TITLE
Request: Catch me up on current Mundana Replication

### ISSUE THREAD

**Issue by arturwhite1** [inspiredlab/inspiredlab.github.io#79]
## Fellow Agiles, 
I have not been around in the meetings. I am wondering what the status is on the replication? I saw the repo MaterialGirl created. Is this where we are providing all the theme files that we create? Thanks @cwalters126 @fedebassetti @leenabusalem @RaviHarsha

**Comment by cwalters126** [comment]
Create a branch in the material girl repo to start editing the mundana theme so it includes our inspired lab info! The material girl repo should be almost a direct duplicate of the wowthemes mundana @arturwhite1

**Comment by fedebassetti** [comment]
@arturwhite1 we also found a simple way to link-in profile pictures and pictures in general, so we don't have to store as local assets if they are available on the web somewhere.
we took rounds at showing where it would apply in the code @leenabusalem 🐝  can now show you what we're talking about. She also showed up her git command line skills right on live camera (though we might need a screen sharing support group). Go @leenabusalem 💪
